libs/pages/elewa/home/src/lib/components/jobs-section/jobs-section.component.scss

```diff
@@ -0,0 +1,24 @@
+.jobs {
+    min-width: 70vw;
+    min-height: 70vh;
+    font-family: var(--elewa-group-website-dmsans-regular);
+    font-size: var(--elewa-group-website-font-size-paragraph);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+  }
+  
+  .jobs a{
+    color: var(--elewa-group-website-color-primary);
+    text-decoration: none;
+    font-family: var(--elewa-group-website-dmsans-bold);
+  }
```

Comment:
The text link needs to have an underline on hover (basic features of a hyperlink)